public/app/plugins/datasource/prometheus/components/PromExploreExtraField.test.tsx

```diff
@@ -1,28 +1,35 @@
 import React from 'react';
-import { shallow } from 'enzyme';
-import { PromExploreExtraField, PromExploreExtraFieldProps } from './PromExploreExtraField';
+import { render, screen } from '@testing-library/react';
```

Comment:
Dipping into the testing lib, nice :)

Reply:
Hahhaha yeah! I really like react testing library. Moreover, have you seen their logo? 😂 🧡